boost-configure

```diff
@@ -0,0 +1,11 @@
+
+./bootstrap.sh "$@" --without-libraries=atomic,chrono,container,context,coroutine,coroutine2,date_time,exception,filesystem,graph,graph_parallel,iostreams,locale,log,math,mpi,program_options,python,random,regex,serialization,signals,system,test,thread,timer,type_erasure,wave
```

Comment:
I wonder if it wouldn't be enough to copy the `boost` directory from the tarball to `/app/include`. Seems to contain all headers-only libraries.